frontend/src/models/User.ts

```diff
@@ -1,25 +1,19 @@
-import { IAuthResponse } from "@/models/IAuthResponse";
-
-export default class User {
-  userName: string = "";
-  name: string = "Login / Sign Up";
-  loggedIn: boolean = false;
-  profileImage: string = "";
-
-  constructor(response?: IAuthResponse) {
-    if (response) {
-      this.loggedIn = true;
-      this.userName = response.user_id;
-
-      for (let claim of response.user_claims) {
-        if (claim.typ.indexOf("/identity/claims/name") > 0) {
-          this.name = claim.val;
-        }
-
-        if (claim.typ.indexOf("profile_image_url_https") > 0) {
-          this.profileImage = claim.val;
-        }
-      }
-    }
-  }
-}
+import jsonwebtoken from "jsonwebtoken";
+
+export default class User {
+  userName: string = "";
+  name: string = "Login / Sign Up";
```

Comment:
I think it would be better for us to move constant strings like "Login / Sign Up" into some constants file that can be easily localized later. I recall in another open hack they have a few different languages supported.

Reply:
Agreed - going to hold off on this for this PR.  We can do a round of localization later.